pkg/render/manager.go

```diff
@@ -264,9 +264,7 @@ func (c *managerComponent) managerDeployment() *appsv1.Deployment {
 // managerVolumes returns the volumes for the Tigera Secure manager component.
 func (c *managerComponent) managerVolumeMounts() []corev1.VolumeMount {
 	if c.cfg.KeyValidatorConfig != nil {
-		trustedVolumeMount := c.cfg.TrustedCertBundle.VolumeMount(c.SupportedOSType())
-		trustedVolumeMount.MountPath = "/etc/ssl/certs/"
```

Comment:
@rene-dekker I am not sure why this mount path is reset to `/etc/ssl` but from my testing manager starts fine.

Reply:
"/etc/ssl/cert.pem" is the default location for Alpine Linux, which the manager is based on.

Reply:
Actually I'm seeing mixed results. Go reads from here for alpine: https://go.dev/src/crypto/x509/root_linux.go

But here I see the suggestion to add it at /etc/ssl/certs/ca-certificates.crt: https://stackoverflow.com/questions/67231714/how-to-add-trusted-root-ca-to-docker-alpine

Reply:
Trusted bundle volume mounts are removed as they don't seem to be used.